backend/src/services/worker/worker.service.ts

```diff
@@ -0,0 +1,89 @@
+import { worker as workerRep } from '~/data/repositories/repositories';
+import {
+  EAMWorkerCreateRequestDto,
+  EAMWorkerResponseDto,
+  TokenPayload,
+} from '~/common/types/types';
+import { Worker as WorkerEntity } from './worker.entity';
+import { HttpCode } from '~/common/enums/http/http';
+import {
+  encrypt as encryptServ,
+  token as tokenServ,
+  master as masterServ,
+} from '~/services/services';
+import { InvalidCredentialsError } from '~/exceptions/invalid-credentials-error/invalid-credentials-error';
+
+type Constructor = {
+  workerRepository: typeof workerRep;
+  encryptService: typeof encryptServ;
+  tokenService: typeof tokenServ;
+  masterService: typeof masterServ;
+};
+
+class Worker {
+  #workerRepository: typeof workerRep;
+  #encryptService: typeof encryptServ;
+  #tokenService: typeof tokenServ;
+  #masterService: typeof masterServ;
+
+  constructor({
+    workerRepository,
+    encryptService,
+    tokenService,
+    masterService,
+  }: Constructor) {
+    this.#workerRepository = workerRepository;
+    this.#encryptService = encryptService;
+    this.#tokenService = tokenService;
+    this.#masterService = masterService;
+  }
+
+  public async getAll(): Promise<Array<EAMWorkerResponseDto>> {
```

Comment:
You should remove all getAll functionality because it is already done in https://github.com/BinaryStudioAcademy/bsa-winter-2021-2022-bws/pull/37